jambalaya-tenancy-spock/src/main/groovy/com/tailrocks/jambalayay/tenancy/spock/ActiveTenantSpock.groovy

```diff
@@ -0,0 +1,14 @@
+package com.tailrocks.jambalayay.tenancy.spock
+
+
+import org.spockframework.runtime.extension.ExtensionAnnotation
+
+import java.lang.annotation.*
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target([ElementType.TYPE, ElementType.FIELD, ElementType.METHOD])
+@ExtensionAnnotation(TenantSpockExtension.class)
+@Inherited
+@interface ActiveTenantSpock {
```

Comment:
Let's rename it to `ActiveTenant`, we already use different package, normally no one will use `ActiveTenant` from jUnit and `ActiveTenant` from Spock inside one test class.

Reply:
Done!